tests/components/kitchen_sink/test_backup.py

```diff
@@ -0,0 +1,162 @@
+"""Test the Kitchen Sink backup platform."""
+
+from collections.abc import AsyncGenerator
+from pathlib import Path
+from unittest.mock import patch
+from uuid import UUID
+
+import pytest
+
+from homeassistant.components.backup import Backup
+from homeassistant.components.kitchen_sink import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.typing import WebSocketGenerator
+
+
+@pytest.fixture(autouse=True)
+async def backup_only() -> AsyncGenerator[None]:
+    """Enable only the backup platform.
+
+    The backup platform is not an entity platform.
+    """
+    with patch(
+        "homeassistant.components.kitchen_sink.COMPONENTS_WITH_DEMO_PLATFORM",
+        [],
+    ):
+        yield
+
+
+@pytest.fixture(autouse=True)
+async def setup_integration(hass: HomeAssistant) -> AsyncGenerator[None]:
+    """Set up Kitchen Sink integration."""
+    with patch("homeassistant.components.backup.is_hassio", return_value=True):
+        assert await async_setup_component(hass, "backup", {"backup": {}})
```

Comment:
Should we instead do `from homeassistant.components.backup import DOMAIN as BACKUP_DOMAIN`?

It's a matter of personal taste though, both work.

Reply:
I'll change it. :+1: